phpcs.xml

```diff
@@ -14,8 +14,8 @@
     <rule ref="Generic.Files.LineEndings" />
     <rule ref="Generic.Files.LineLength">
         <properties>
-            <property name="lineLimit" value="125" />
-            <property name="absoluteLineLimit" value="125" />
+            <property name="lineLimit" value="119" />
+            <property name="absoluteLineLimit" value="119" />
```

Comment:
That naming is really confusing. I also thought of numbers of lines in a file immediately :S Btw do we also have a limit for that number?